src/js/components/TextArea/doc.js

```diff
@@ -35,3 +35,35 @@ Only use this when the containing context provides sufficient affordance.`,
 
   return DocumentedTextArea;
 };
+
+export const themeDoc = {
+  'global.colors.placeholder': {
+    description: 'The placeholder color used for TextArea.',
+    type: 'string',
+    defaultValue: '#AAAAAA',
+  },
+  'global.control.border.width': {
+    description: 'The border width.',
+    type: 'string',
+    defaultValue: '1px',
+  },
+  'global.input.weight': {
+    description: 'The font weight of the label.',
```

Comment:
I don't think TextArea has a "label". I suspect this means the font weight of the text entered?

Reply:
working on my English ;)